packages/shopify-codemod/test/fixtures/conditional-assign-to-if-statement/assign-to-other-member.output.js

```diff
@@ -0,0 +1,3 @@
+this.foo || (this.bar = 'lol');
+self.foo || (self.bar = 'lols');
+x || (y = 'lols');
```

Comment:
This is possible, but I was thinking of doing it in another transformation.  The intent of this was to replace code generated from CS like `fakeLink ||= @document.createElement('a')`.